core/object/object.cpp

```diff
@@ -865,7 +865,9 @@ String Object::to_string() {
 		}
 	}
 	if (_extension && _extension->to_string) {
-		return _extension->to_string(_extension_instance);
+		String ret;
+		_extension->to_string(_extension_instance, &ret);
+		return ret;
```

Comment:
That's definitely a better, original code won't handle UTF-8 correctly and parse it as ASCII.

Note: `godot-cpp` seems to be ignoring this, so it does not require any changes except header update after this.

https://github.com/godotengine/godot-cpp/blob/5eb1c54116a135ac3d7f80b50cbe3cbe137318c6/include/godot_cpp/core/class_db.hpp#L156